code/modules/jobs/job_types/chief_engineer.dm

```diff
@@ -28,6 +28,18 @@
 
 	family_heirlooms = list(/obj/item/clothing/head/hardhat/white, /obj/item/screwdriver, /obj/item/wrench, /obj/item/weldingtool, /obj/item/crowbar, /obj/item/wirecutters)
 
+	mail_goodies = list(
+		/obj/item/stack/sheet/mineral/diamond{amount = 2} = 15,
+		/obj/item/stack/sheet/mineral/uranium{amount = 5} = 15,
+		/obj/item/stack/sheet/mineral/plasma{amount = 5} = 15,
+		/obj/item/stack/sheet/mineral/gold{amount = 2} = 15,
+		/obj/item/food/cracker = 25, //you know. for poly
+		/obj/item/wrench/caravan = 3,
+		/obj/item/wirecutters/caravan = 3,
+		/obj/item/screwdriver/caravan = 3,
+		/obj/item/crowbar/red/caravan = 3
```

Comment:
is there an appropriate kind of welder that could go with these, or would that be just kind of a dud gift because of the station's ability to print experimental welders later on?

Reply:
Honestly, not really. The closest alternative I could think of would be the plasma cutter, but I think this is decent to leave as is for now.